target-app/Networking/Services/Endpoints/TargetEndpoint.swift

```diff
@@ -16,12 +16,13 @@ internal enum TargetEndpoint: RailsAPIEndpoint {
         radius: Double,
         topicId: Int
     )
+    case get
```

Comment:
This as an endpoint identifier could be confusing, maybe `getAll` or `fetchAll`

Reply:
done!